src/rp2040.ts

```diff
@@ -1485,10 +1485,10 @@ export class RP2040 {
       const operand2 = this.registers[Rm] + (this.C ? 0 : 1);
       const result = (operand1 - operand2) | 0;
       this.registers[Rdn] = result;
-      this.N = (operand1 | 0) < (operand2 | 0);
+      this.N = !!(result & 0x80000000);
```

Comment:
Why not `result < 0`? I think it's a bit easier to read it that way (and we also calculate `N` like that in other places, such as the `CMP` instruction